src/api_routes_http.rs

```diff
@@ -293,93 +278,55 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
-          .route(web::post().to(reset_password)),
-      )
-      .service(
-        // Handle captcha separately
-        web::resource("/user/get_captcha")
-          .wrap(rate_limit.post())
-          .route(web::get().to(get_captcha)),
-      )
-      .service(
-        web::resource("/user/export_settings")
-          .wrap(rate_limit.import_user_settings())
-          .route(web::get().to(export_settings)),
+          .route("register", web::post().to(register))
+          .route("login", web::post().to(login))
+          .route("password_reset", web::post().to(reset_password))
+          .route("get_captcha", web::get().to(get_captcha))
+          .route(
+            "/password_change",
+            web::post().to(change_password_after_reset),
+          )
+          .route("/change_password", web::put().to(change_password))
+          .route("/totp/generate", web::post().to(generate_totp_secret))
+          .route("/totp/update", web::post().to(update_totp)),
       )
       .service(
-        web::resource("/user/import_settings")
+        web::resource("/account/export")
           .wrap(rate_limit.import_user_settings())
+          .route(web::get().to(export_settings))
           .route(web::post().to(import_settings)),
       )
-      // TODO, all the current account related actions under /user need to get moved here eventually
       .service(
         web::scope("/account")
-          .wrap(rate_limit.message())
-          .route("/list_media", web::get().to(list_media)),
-      )
-      // User actions
-      .service(
-        web::scope("/user")
-          .wrap(rate_limit.message())
-          .route("", web::get().to(read_person))
+          .route("/list_media", web::get().to(list_media))
           .route("/mention", web::get().to(list_mentions))
-          .route(
-            "/mention/mark_as_read",
-            web::post().to(mark_person_mention_as_read),
-          )
           .route("/replies", web::get().to(list_replies))
-          // Admin action. I don't like that it's in /user
-          .route("/ban", web::post().to(ban_from_site))
-          .route("/banned", web::get().to(list_banned_users))
           .route("/block", web::post().to(block_person))
-          // TODO Account actions. I don't like that they're in /user maybe /accounts
           .route("/logout", web::post().to(logout))
           .route("/delete_account", web::post().to(delete_account))
           .route(
-            "/password_change",
-            web::post().to(change_password_after_reset),
+            "/mention/mark_as_read",
+            web::post().to(mark_person_mention_as_read),
           )
-          // TODO mark_all_as_read feels off being in this section as well
           .route(
-            "/mark_all_as_read",
+            "/mention/mark_all_as_read",
             web::post().to(mark_all_notifications_read),
           )
           .route("/save_user_settings", web::put().to(save_user_settings))
```

Comment:
Maybe just `save` ? I dunno.

Reply:
Made it `/account/settings/save`

Reply:
This one too, burying it under settings doesn't seem right, when all the other actions aren't under settings either.

Reply:
Its specifically about settings so it makes sense (same place as import/export). And buried is the wrong word, its still easy to find in docs etc.